### PR TITLE
Add docker-registry example package

### PR DESCRIPTION
**Pull request by greghaynes**

### CODE REVIEW

**Comment by nimakaviani** [comment]
very cool. to some extent addresses this issue as well: https://github.com/cnoe-io/idpbuilder/issues/49

**Comment by nabuskey** [comment]
Can you add a readme similar to other examples we have?

**Comment by greghaynes** [comment]
@jessesanford I agree with @nabuskey we should add a readme. One thing I would definitely include in there is this: https://engineering.autodesk.com/summits/techx2024/proposals/view/87157 - we set a default password and what it is.

**Review by csantanapr — CHANGES_REQUESTED** [review]
Please a hello world in a `README.md` at the root of the new example folder

File: examples/ref-implementation/backstage/manifests/install.yaml [hunk 1]
```diff
@@ -166,7 +166,7 @@ data:
       locations:
         # Examples from a public GitHub repository.
         - type: url
-          target: https://gitea.cnoe.localtest.me:8443/giteaAdmin/idpbuilder-localdev-backstage-entities/src/branch/main/catalog-info.yaml
+          target: https://gitea.cnoe.localtest.me:8443/giteaAdmin/idpbuilder-localdev-backstage-templates-entities/raw/branch/main/catalog-info.yaml
```
Comment (csantanapr) [inline examples/ref-implementation/backstage/manifests/install.yaml:169]: rebase this is already fix in main
https://github.com/cnoe-io/idpbuilder/commit/ea9c56e6231c05cc33ed6b33a00bec79008a8a16

Comment (jessesanford) [reply examples/ref-implementation/backstage/manifests/install.yaml:169]: Yes this is the cherry-pick of that exact commit to this branch I needed it to make sure things were working correctly. The merge to main should be noOp here. Happy to remove it if that is more appropriate.

**Comment by omrishiv** [comment]
Do we want to use gitea's container registry or maybe [harbor](https://github.com/goharbor/harbor)? gitea's would mean not having to deploy an extra resource into the cluster whereas harbor provides supply chain security and image scanning, which could be a nice addition

**Comment by cmoulliard** [comment]
> Do we want to use [gitea's container registry](https://docs.gitea.com/usage/packages/container) or maybe [harbor](https://github.com/goharbor/harbor)? gitea's would mean not having to deploy an extra resource into the cluster whereas harbor provides supply chain security and image scanning, which could be a nice addition

As the idpbuilder's gitea server can also be used as images registry, such a package is not needed OR could be proposed as separated ref implementation example

**Comment by nimakaviani** [comment]
yeah closing this as #282 takes care of this. Feel free to reopen if you think a separate solution is needed.

**Pull request closed without merging**